trade-finance-manager-ui/server/routes/case/index.js

```diff
@@ -121,7 +122,7 @@ router.get('/:_id/underwriting/lead-underwriter/assign', underwritingController.
 router.post('/:_id/underwriting/lead-underwriter/assign', underwritingController.postAssignLeadUnderwriter);
 
 router.get('/:_id/underwriting/managers-decision/edit', underwritingController.getUnderwriterManagersDecisionEdit);
-router.post('/:_id/underwriting/managers-decision/edit', underwritingController.postUnderwriterManagersDecision);
+router.post('/:_id/underwriting/managers-decision/edit', xss(), underwritingController.postUnderwriterManagersDecision);
```

Comment:
I've only added this sanitising to this specific endpoint rather than globally - I thought it's likely there would be unwanted side-effects if I used it everywhere.

Reply:
Can we check other `POST` endpoints where `xss()` might also be required please.

Reply:
Please could we add a separate ticket to cover this and removing the other usages of the `| safe` filter? I'm happy to pick up that ticket next.

I think there'll be a lot of extra QA testing needed if we're changing lots of endpoints which I think will end up being quite confusing. Also it'll be quite time consuming going through all the endpoints to work out where this is needed so it would be good to not delay getting the rest of this ticket deployed.

Alternatively, we could add the xss filtering globally. It would take a lot less dev time than going through the endpoints individually but would need a lot of smoke testing. Again, if we go for this approach I think it should be as a separate ticket to make it easier to track.

Let me know what you think.

Reply:
I can create a separate ticket for `safe` filter removal.

Reply:
Let's create a separate ticket. Otherwise this PR can get quite big